reference/docs-conceptual/dsc/resources/authoringResourceComposite.md

```diff
@@ -183,6 +181,28 @@ configuration RenameVM
 }
 ```
 
+You can also use this resource to create multiple VMs by passing in an array of VM names to the xVirtualMachine resource.
+
+```PowerShell
+    Configuration MultipleVms
+    {
+        Import-DscResource -Module xVirtualMachine
+        Node localhost
+        {
+            xVirtualMachine VMs
+            {
+                VMName = "IIS01", "SQL01", "SQL02"
+                SwitchName = "Internal"
+                SwitchType = "Internal"
+                VhdParentPath = "C:\Demo\VHD\RTM.vhd"
+                VHDPath = "C:\Demo\VHD"
+                VMStartupMemory = 1024MB
+                VMState = "Running"
+            }
+        }
+    }
+```
+
 ## Supporting PsDscRunAsCredential
 
 >**Note:** **PsDscRunAsCredential** is supported in PowerShell 5.0 and later.
```

Comment:
Not part of your change, but if you'd also fix the note since it renders incorrectly, would appreciate it.

Reply:
Makes sense. Fixed